frontend/src/components/Pokemon/PokemonCard/PokemonCard.tsx

```diff
@@ -0,0 +1,21 @@
+import styles from './PokemonCard.module.css'
+import { Link } from 'react-router-dom'
+import { PokemonProps } from '../domain/buisness_objetcs/pokemon.type'
+
+const Pokemon = ({ name, id, weight, height }: PokemonProps) => {
+   return (
+      <Link to={`/pokemon/${id}`}>
+         <div className={styles.card}>
+            <h3>{name}</h3>
+            <img src={`https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/showdown/${id}.gif`} alt={`pokemon_${id}`} />
```

Comment:
- <!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
    
    **🎯 suggestion (non-blocking):** <subject>You can set the URL in a constant (for modularity)
    
    <discussion>